core/domain/feedback_jobs_continuous_test.py

```diff
@@ -696,3 +697,83 @@ def test_realtime_with_batch_computation(self):
                     'num_open_threads': 2,
                     'num_total_threads': 2,
                 })
+
+    def test_open_fixed_thread(self):
+        with self._get_swap_context():
+            # Create test objects.
+            user_id = 'uid'
+            exp_id = 'eid'
+            self.save_new_valid_exploration(exp_id, 'owner')
+
+            event_services.FeedbackThreadStatusChangedEventHandler.record(
+                exp_id, feedback_models.STATUS_CHOICES_FIXED,
+                feedback_models.STATUS_CHOICES_OPEN)
+
+            # Start job.
+            self.process_and_flush_pending_tasks()
+            MockFeedbackAnalyticsAggregator.start_computation()
+            self.assertEqual(
+                self.count_jobs_in_taskqueue(
+                    taskqueue_services.QUEUE_NAME_CONTINUOUS_JOBS), 1)
+            self.process_and_flush_pending_tasks()
+
+            # Stop job.
+            MockFeedbackAnalyticsAggregator.stop_computation(user_id)
+            self.assertEqual(
+                self.count_jobs_in_taskqueue(
+                    taskqueue_services.QUEUE_NAME_CONTINUOUS_JOBS), 0)
+
+            self._flush_tasks_and_check_analytics(
+                exp_id, {
+                    'num_open_threads': 0,
+                    'num_total_threads': 0,
+                })
+
+            # Create another thread but don't start job.
```

Comment:
Not sure why you need all this.

Reply:
this is to show that num_open_threads increments when we create a new thread

Reply:
OK, but how is it relevant to what you're testing in this particular test?

Reply:
Removed